### ISSUE TITLE
remove shiny material inside chamber

### ISSUE THREAD

**Issue by untzag** [uw-madison-chem-shops/wisconsin-photoreactor#43]
I don't believe that the shiny material makes an effective contribution to the photoreactor. It's just yet another part that needs to be sourced and the adhesive is prone to failure. I move that we remove it from our documentation and recommendations.

@plampkin thoughts?

**Comment by plampkin** [comment]
My sense is that you're likely correct. I suspect the vast majority of the performance comes from LED choice and positioning, not the presence or absence of mirrored material. I also agree with your dislike of the poor adhesive/performance of the vinyl mirrored material we currently use -- it's very inelegant relative to the rest of our reactor.

One thing that I've thought about is moving to shiny metal tape (metalized mylar or foil duct tape) that'd be easier to work with, more reflective and longer lasting. Such materials will have part numbers we can refer to and would likely perform better. I suggest we produce a reaction chamber and run a comparison test using this such a material before we totally remove it from our documentation.

What do you think?

**Comment by untzag** [comment]
That makes sense. Are you willing to do that work @plampkin?

**Comment by plampkin** [comment]
Absolutely! Do you have any metalized Mylar or foiled duct tape down in the shop? I can run the comparison over the next few days.

**Comment by untzag** [comment]
Nope, I don't think we have anything. Might be worth a look around.

**Comment by plampkin** [comment]
I'll order some of the following then:

Manufacturer: 3M
Part#: 1-5-438

Digikey: https://www.digikey.com/en/products/detail/3m-(tc)/1-5-438/2649729
Newark: https://www.newark.com/3m/1-5-438/tape-4-57m-x-25-4mm-silver-alum/dp/99Y5560

This reflective metal foil has everything we need in terms of documentation!
